README.md

```diff
@@ -1,6 +1,9 @@
 
 [![Build Status](https://dev.azure.com/ms/azure-databricks-operator/_apis/build/status/microsoft.azure-databricks-operator?branchName=master)](https://dev.azure.com/ms/azure-databricks-operator/_build/latest?definitionId=254&branchName=master)
 
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/microsoft/azure-databricks-operator)](https://goreportcard.com/report/github.com/microsoft/azure-databricks-operator)

```

Comment:
Can we add this to the same line as the build status badge above? Looks a little nicer when they're on the same line.

Reply:
very good idea, done :)